public/js/scripts.js

```diff
@@ -590,7 +590,7 @@ function vote(type, id, dir) {
 
 function submitFeedback(){
     var feedback = $(".feedbackTextarea").val();
-    $.post("/api/feedback_comment",{feedback_comment: feedback},function(data){
+    $.post("/api/feedback",{feedback: feedback},function(data){
```

Comment:
Just a suggestion, not critical...

The URL is /api/feedback, the data being passed is (yes, feedback) a comment.  Call your feedback comment `comment`.  My biggest reason why is to help distinguish between the Feedback Controller and the data passed to the Feedback Controller.  This suggestion is mostly just to aid in the clarity.  If we use "feedback" everywhere, it becomes more difficult to understand which name is being used for what purpose.  Speaking from years of experience, I have been burned hard a number of times because I have chosen a name that was difficult to discern what *specifically* I was referring to it any given instance.

An example, I created an app to web scrape daily images from the web.  There's the date that the scraping is happening, also the date of the artifact being scraped.  That's two different dates.  Call them both "date" and things become confusing real quick!!